lib/routes/user/received_invoice_notification.dart

```diff
@@ -72,7 +92,9 @@ class InvoiceNotificationsHandler {
 
     _sentPaymentResultSubscription =
         _accountBloc.fulfilledPayments.listen((fulfilledPayment) {
-      showFlushbar(_context, message: "Payment was successfuly sent!");
+          // Trigger the collapse animation and show flushbar 200ms after the animation is completed
+          paymentRequest.key.currentState.controller.reverse();
```

Comment:
See above comment, if we will move the logic into the dialog itself, this static global key access won't be needed.